src/DashboardService.Data/BoardRepository.cs

```diff
@@ -18,28 +22,104 @@ public BoardRepository(IDataProvider provider)
     _provider = provider;
   }
 
-  public Task CreateAsync(DbBoard board)
+  public async Task<Guid?> CreateAsync(DbBoard board)
   {
-    throw new NotImplementedException();
+    _provider.Boards.Add(board);
+
+    await _provider.SaveAsync();
+
+    return board.Id;
   }
 
-  public Task<bool> EditByIdAsync(Guid id, JsonPatchDocument<DbChangeLog> dbChangeLog)
+  public async Task<bool> EditByIdAsync(Guid id, Guid modifiedById, JsonPatchDocument<DbBoard> request, CancellationToken ct)
   {
-    throw new NotImplementedException();
+    DbBoard board = await _provider.Boards.FirstOrDefaultAsync(x => x.Id == id, ct);
+
+    if (board is null || request is null)
```

Comment:
your request supposed to pass validation

Reply:
++